docs/src/model_docs/params_vertical.md

```diff
@@ -116,6 +116,7 @@ specific_leaf = "Sl"
 | **`leaf_area_index`** | leaf area index | m``^2`` m``{-2}`` | - |
 | `waterlevel_land` | water level land | mm | - |
 | `waterlevel_river` | water level river | mm | - |
+| `total_storage` | total water storage (excluding floodplains) | mm | - |
```

Comment:
Since you mention that we exclude floodplains, good to mention that lakes and reservoirs are also not included

Reply:
Good point, I'll include that.